src/GoodsList.tsx

```diff
@@ -6,9 +6,16 @@ type Props = {
 };
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
-  <ul>
+  <ul className="goodsList">
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+        className="good"
+        style={{
+          color: `${good.color}`,
```

Comment:
```suggestion
          color: good.color,
```

Useless template string